### PR TITLE
Document decision points

### LINKED ISSUE #32: Document decision points in the WNV phylogenetic build
### Possible solution
Add information to the README file about the decisions points that were made to build the global and the local WNV builds:

- Selection of roots
- Subsampling
- Lineage calling in Nextclade and coloring in Auspice

Then, select what points to highlight in the Description section of the UI (description.md)

### PR DESCRIPTION
**Pull request by DOH-LMT2303**
## Description of proposed changes

Document decision points

## Related issue(s)

* https://github.com/nextstrain/WNV/issues/32

## Checklist

- [x] Checks pass

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,29 +23,39 @@ auspice -help
...
+Or execute the Washington focused build 
+```bash
+nextstrain build phylogenetic --configfile build-configs/washington-state/config.yaml
+```
 Inside the phylogenetic folder there should be at least one output file: WNV-nextstrain_NA.json
 
-Run the build all at once
+Run the build all at once. This option defaults to the global build.
 ```bash
 nextstrain build
```
Comment (DOH-LMT2303) [inline README.md]: Update instruction to run: nextstrain build phylogenetic

File: README.md [hunk 2]
```diff
@@ -65,6 +65,7 @@ _Mencattelli, G., Ndione, M.H.D., Silverj, A. et al. Spatial and temporal dynami
...
 _Hadfield J, Brito AF, Swetnam DM, Vogels CBF, Tokarz RE, Andersen KG, Smith RC, Bedford T, Grubaugh ND. Twenty years of West Nile virus spread and evolution in the Americas visualized by Nextstrain. PLoS Pathog. 2019 Oct 31;15(10):e1008042. doi: 10.1371/journal.ppat.1008042. PMID: 31671157; PMCID: PMC6822705._
 
-
+### Subsampling
+The Washington focused WNV build pulls all the WNV sequences available in NCBI and filters the data in the phylogenetic workflow based on criteria defined in the config.yml file that is located inside the defaults folder. The subsampling criteria focuses on geographic location selecting all sequences from Washington, neighboring states, and region but up to a maximum of 5,000 sequences; and up to 300 sequences selected randomly from the rest of the states. All sequences have to meet a minimum genome length that is also specified as part of the subsampling criteria. There is more information about how to subsample data in Nextstrain here [Filter and Subsampling](https://docs.nextstrain.org/en/latest/guides/bioinformatics/filtering-and-subsampling.html)
```
Comment (DOH-LMT2303) [inline README.md]: Update the config file location

File: README.md (hunk 1 above)
Comment (DOH-LMT2303) [reply README.md]: Resolved with https://github.com/nextstrain/WNV/pull/51/commits/7ea2f9ad639a4e34a7cfb3552f721d336f992a4c

File: README.md (hunk 2 above)
Comment (DOH-LMT2303) [reply README.md]: Resolved with https://github.com/nextstrain/WNV/pull/51/commits/76522ea683f822e5ea0c1c19c227b378171ba6e1

**Review by j23414 — APPROVED** [review]
Looks good to me! And we walked through the text together.

**Pull request merged**